VOKMockUrlProtocol.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "VOKMockUrlProtocol"
-  s.version          = "2.2.0"
+  s.version          = "2.2.1"
```

Comment:
This is going to need to be a major version bump, since it'll break compatibility with the existing naming format.